storageapi/src/vespa/storageapi/mbusprot/protobuf/common.proto

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+syntax = "proto3";
+
+option cc_enable_arenas = true;
+
+package storage.mbusprot.protobuf;
+
+// Note: we use a *Request/*Response naming convention rather than *Command/*Reply,
+// as the former is the gRPC convention and that's where we intend to move.
+
+// Next tag to use: 2
```

Comment:
Re discussion: Remove next tag comments in favor of `reserved` if / when removing a field.

Reply:
Done